flash-cards.rb

```diff
@@ -0,0 +1,135 @@
+class Flashcard
+  attr_accessor :front, :back
+
+  @@all_flashcards = []
+  $flashcards = @@all_flashcards
```

Comment:
Think this global `$flashcards` is unnecessary. You could have a class method that returns all the instances of the Flashcards (that is to say, a method that returns `@@all_flashcards`).